tick/inference/hawkes_conditional_law.py

```diff
@@ -367,6 +371,17 @@ def _init_lambdas(self):
         self.mean_intensity = [0] * self.n_nodes
         self._lam_N = [0] * self.n_nodes
         self._lam_T = [0] * self.n_nodes
+        # Used to store the number of events of each component that
+        # have been used to perform estimation on all the lags
+        # versus the number of events that could not be used for all the lags
+        # Warning : we don't take care of marks for this computation
+        # normally we should do this computation independantly for each mark
```

Comment:
yes it should be still compatible, this is just about the test used to control if a component has at least some usable events for estimation.